inc/layers.php

```diff
@@ -869,16 +869,7 @@ function get_layer($post_id = false) {
         $excerpt = apply_filters('translate_text', $post->excerpt, odm_language_manager()->get_current_language());
 
         $in_category = get_the_terms( $post->ID, 'layer-category' );
-        if (($key = array_search('map-catalogue', $in_category)) !== false) {
-            unset($array[$key]);
-        }
-        foreach ($in_category as $key => $in_cat) {
-          if($in_cat->slug == "map-catalogue"){
-            unset($in_category[$key]);
-          }
-        }
-        $in_category = array_values($in_category);
-
+        
```

Comment:
@Huyeng you added this code recently as part of https://github.com/OpenDevelopmentMekong/wp-odm_theme/commit/38684455e683d84f4fec6ec83ac577ad4a8d2c30 (3 days ago).

What is the reason for removing this code now?